### PR TITLE
chore(docs): Add model reference documentation

### PR DESCRIPTION
**Pull request by lekaf974**
**Notes for Reviewers**

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.
 

Loaclly tested
image

### CODE REVIEW

File: docs/_data/mesheryctlcommands/cmds.yml [hunk 1]
```diff
@@ -197,7 +197,7 @@ system:
           name: --skip-browser
           description: (optional) skip opening of MesheryUI in browser.
           usage: mesheryctl system dashboard --skip-browser
-          example: mesheryctl system dashboard --skip-browser 
+          example: mesheryctl system dashboard --skip-browser
```
Comment (leecalcote) [inline docs/_data/mesheryctlcommands/cmds.yml:200]: Good catch!

File: docs/_data/mesheryctlcommands/cmds.yml [hunk 2]
```diff
@@ -843,7 +843,7 @@ registry:
           description: id of the spreadsheet to download
 model:
   name: model
-  description: 
+  description: A unit of packaging to define managed infrastructure and their relationships, and details specifics of how to manage them.
```
Comment (leecalcote) [inline docs/_data/mesheryctlcommands/cmds.yml:846]: Perfect.

File: docs/pages/reference/mesheryctl-commands.md [hunk 3]
```diff
@@ -744,6 +744,48 @@ Installation, troubleshooting and debugging of Meshery and its adapters
 </thead>
 </table>
 
+
+## Meshery Models Registered
```
Comment (leecalcote) [inline docs/pages/reference/mesheryctl-commands.md:748]:
```suggestion
## Meshery Models
```

**Review by leecalcote — APPROVED** [review]

**Review by leecalcote — APPROVED** [review]

File: docs/_data/mesheryctlcommands/cmds.yml (hunk 1 above)
Comment (lekaf974) [reply docs/_data/mesheryctlcommands/cmds.yml:200]: Thanks to my IDE

**Pull request merged**